web/src/app/modules/views/main/editors/modules/folder-details/components/folder-details.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, OnInit } from '@angular/core';
+import { SpecmateViewBase } from '../../../base/specmate-view-base';
+import { SpecmateDataService } from '../../../../../../data/modules/data-service/services/specmate-data.service';
+import { NavigatorService } from '../../../../../../navigation/modules/navigator/services/navigator.service';
+import { ActivatedRoute } from '@angular/router';
+import { ConfirmationModal } from '../../../../../../notification/modules/modals/services/confirmation-modal.service';
+import { TranslateService } from '@ngx-translate/core';
+import { IContainer } from '../../../../../../../model/IContainer';
+import { Folder } from '../../../../../../../model/Folder';
+import { CEGModel } from '../../../../../../../model/CEGModel';
+import { Process } from '../../../../../../../model/Process';
+import { Type } from '../../../../../../../util/type';
+import { Requirement } from '../../../../../../../model/Requirement';
+
+@Component({
+    moduleId: module.id.toString(),
+    selector: 'folder-details',
+    templateUrl: 'folder-details.component.html',
+    styleUrls: ['folder-details.component.css']
+})
+
+export class FolderDetails extends SpecmateViewBase {
+    cegModelType = CEGModel;
+    processModelType = Process;
+    folderType = Folder;
+    reqType = Requirement;
+
+    /** Constructor */
+    constructor(
+        dataService: SpecmateDataService,
+        navigator: NavigatorService,
+        route: ActivatedRoute,
+        modal: ConfirmationModal,
+        translate: TranslateService) {
+        super(dataService, navigator, route, modal, translate);
+    }
+
+    private folder: Folder;
+    private contents: IContainer[];
+    private transitiveContents: IContainer[];
+
+    private async updateTransitiveContent() {
```

Comment:
we don't need the transitive folder contents. Direct folders are enough. I clarified this in the trello card